easybuild/tools/options.py

```diff
@@ -404,6 +405,11 @@ def override_options(self):
                            'strlist', 'extend', None),
             'filter-env-vars': ("List of names of environment variables that should *not* be defined/updated by "
                                 "module files generated by EasyBuild", 'strlist', 'extend', None),
+            'filter-rpath-sanity-libs': ("List of libraries that you want to ignore when the RPATH sanity check is "
```

Comment:
`that you want ignore` -> `that should be ignored`

Reply:
Yes, better, will fix